dataeng/processed_data/process_data.py

```diff
@@ -0,0 +1,208 @@
+import csv
+import os
+import time
+
+
+class processing:
+
+    def __init__(self, source_data_path: str, processed_data_path: str):
+        """
+        Source_data_path is the path where source data are located
+        """
+        self.source_data_path = source_data_path
+        self.processed_data_path = processed_data_path
+
+    def get_age(self, birth_date: float):
+        birth_date = float(birth_date)
+        current_time = time.time() * 1000
+        return (current_time - birth_date) / (3.154 * 10 ** 10)
```

Comment:
What is the magic number? 
`3.154 * 10 ** 10`
My suggestion is to store such values as constants. For example,
`MY_MAGIC_NUMBER = 3.154 * 10 ** 10`